packages/execution-environments/src/iframe/IFrameSnapExecutor.ts

```diff
@@ -5,6 +5,13 @@ import { BaseSnapExecutor } from '../common/BaseSnapExecutor';
 import { SNAP_STREAM_NAMES } from '../common/enums';
 
 export class IFrameSnapExecutor extends BaseSnapExecutor {
+  /**
+   * Initialize the IFrameSnapExecutor. This creates a post message stream from
```

Comment:
This also sets up the listeners for incoming requests, binds it to the correct handlers etc via the base constructor. See `BaseSnapExecutor`.

Reply:
Is that in scope for this function documentation?